frontend/app/src/components/status/notifications/NotificationPopup.vue

```diff
@@ -20,22 +20,24 @@ const dismissAll = async () => {
   set(visibleNotification, { ...get(visibleNotification), display: false });
 };
 
-watch(
-  queue,
-  () => {
-    const data = [...get(queue)];
-    if (!get(visibleNotification).display && data.length > 0) {
-      const next = data.shift();
-      if (!next) {
-        return;
-      }
-      nextTick(() => {
-        set(visibleNotification, next);
-      });
+const checkQueue = () => {
+  const data = [...get(queue)];
+  if (!get(visibleNotification).display && data.length > 0) {
+    const next = data.shift();
+    if (!next) {
+      return;
     }
-  },
-  { deep: true }
-);
+    nextTick(() => {
+      set(visibleNotification, next);
+    });
+  }
+};
+
+watch(queue, checkQueue, { deep: true });
+
+onMounted(() => {
+  checkQueue();
+});
```

Comment:
This should fix the issue where the popup of notification for the evm address migration, appear late.